src/analyze.proto

```diff
@@ -32,9 +42,24 @@ message AnalyzeResponse{
   string requestId = 2;
 }
 
+// RecognizersRequest represents the request to get the recognizers
+// known to the analyze service that support the input language via GRPC 
+message RecognizersAllRequest {
+  // The supported language code, in ISO-639 format, https://en.wikipedia.org/wiki/ISO_639-1
+  string language = 3;
+}
+
+// RecognizersResponse represents the analyze service response
+message RecognizersAllResponse{
```

Comment:
```suggestion
message RecognizersGetAllRequest{
```
This would be the same as the call in the recognizers store: https://github.com/microsoft/presidio-genproto/blob/cecff12b0ae1c950a8aa62016760e97030208acb/src/recognizers_store.proto#L54

Reply:
tried it as well, but in proto you cannot define duplicate method names to types unless they are not in the same package. see this similar issue: https://github.com/protocolbuffers/protobuf/issues/3272

Reply:
I think the solution would be to rename the recognizer store api name to CustomRecognizersGetAllRequest but this could be done in a following PR

Reply:
Otherwise we'll have two APIs who sound the same but actually return different results

Reply:
omri374 why call this essage Request , when its a result.
so what did you decide here balteravishay

Reply:
i agree with omri. i will change it as part of this pr.

Reply:
talked with omri, this is a bigger change than suggested and will open a user story for it.